docs/locale/ko/LC_MESSAGES/header/header.po

```diff
@@ -0,0 +1,210 @@
+msgstr "프로젝트 선택기"
+
+#: ../../header/header.rst:19
+msgid ""
+"Users can switch between projects using the project selector provided in "
+"the top bar. By default, the project that user currently belongs to is "
+"selected. Since each project may have different resource policies, "
+"switching projects may also change the available resource policies."
+msgstr "사용자는 상단 바의 프로젝트 선택기를 통하여 사용자의 현재 프로젝트를 선택할 수 있습니다. "
+"각 프로젝트 별로 다른 자원 정책을 가질 수 있으므로, 프로젝트를 변경할 경우 가용 가능한 자원 정책이 변경될 수 있습니다. "
+
+#: ../../header/header.rst:-1 ../../header/header.rst:24
+msgid "Notification"
+msgstr "이벤트 알림"
+
+#: ../../header/header.rst:28
+msgid ""
+"The bell shape button is the event notification button. Events that need "
+"to be recorded during WebUI operation are displayed here. When background"
+" tasks are running, such as creating a compute session, you can check the"
+" jobs here. When the background task is finished."
+msgstr "종 모양의 버튼은 이벤트 알림 버튼입니다. WebUI 동작 과정에서 기록이 필요한 이벤트 내역이 여기에 표시됩니다. "
+"예를 들어, 연산 세션의 생성을 위한 백그라운드 작업이 진행되고 있는 경우, 이벤트 알림에서 해당 내용을 확인할 수 있습니다. "
+
+#: ../../header/header.rst:-1 ../../header/header.rst:39
+msgid "Theme mode"
+msgstr "테마 모드"
+
+#: ../../header/header.rst:43
+msgid ""
+"You can change the theme mode of the WebUI via the dark mode icon on the "
+"right side of the header."
+msgstr "상단 바 우측에 있는 다크모드 아이콘을 통하여 WebUI의 테마를 변경할 수 있습니다. "
+
+#: ../../header/header.rst:52
+msgid "Help"
+msgstr "도움말"
+
+#: ../../header/header.rst:56
+msgid ""
+"There is a question mark icon at the lower right side of the header. "
+"Click this icon to access the web version of this guide document."
+msgstr "상단 바 우측의 물음표 버튼을 통하여, 본 가이드 문서의 웹 버전에 접속할 수 있습니다. "
+
+#: ../../header/header.rst:60
+msgid "User Menu"
+msgstr "사용자 메뉴"
+
+#: ../../header/header.rst:64
+msgid ""
+"Click the person icon on the right side of the top bar to see the user "
+"menu. Each menu item has the following functions."
+msgstr "상단 바 우측의 사람 아이콘 버튼을 클릭하여, 사용자 메뉴를 확인할 수 있습니다. "
+"각 항목은 다음과 같은 기능을 가집니다. "
+
+#: ../../header/header.rst:-1
+msgid "user drop down"
+msgstr ""
+
+#: ../../header/header.rst:72
+msgid ""
+"About Backend.AI: Displays information such as version of Backend.AI "
+"WebUI, license type, etc."
+msgstr "Backend.AI에 대하여: Backend.AI Web-UI의 버전, 라이선스 종류 등과 같은 정보를 표시 합니다. "
+
+#: ../../header/header.rst:74
+msgid "My Account: Check/Update information of current login user"
+msgstr "사용자 정보 변경: 현재 로그인된 사용자 정보를 확인/변경합니다. "
+
+#: ../../header/header.rst:75
+msgid "Preferences: Go to user settings page"
+msgstr "설정: 사용자 설정 페이지로 이동합니다 "
+
+#: ../../header/header.rst:76
+msgid ""
+"Logs / Errors: Go to the log page. You can check the log and error "
+"history recorded on the client side."
+msgstr "로그 / 에러기록: 로그 페이지로 이동합니다. 클라이언트 측에 기록된 로그 및 오류 내역을 확인할 수 있습니다. "
+
+#: ../../header/header.rst:78
+msgid ""
+"Log Out: Log out. Users can also logout by clicking the icon on the far "
+"right of the top bar."
+msgstr "로그아웃: 로그아웃 합니다. 로그아웃은 상단 바의 가장 우측의 아이콘을 클릭해도 할 수 있습니다. "
```

Comment:
We can only log out from the user menu. Therefore, the description for logging out is simply "Log out." I'm not sure if we really need to include this. It seems like it could be removed.

Reply:
I don't think the logout description is necessary, so I'll remove it.